### PR TITLE
Stop using 'deprecated'

### PR DESCRIPTION
**Pull request by martinthomson**
image

### CODE REVIEW

**Review by torgo — APPROVED** [review]
I'm good with this.

**Review by hadleybeeman — APPROVED** [review]
I'm fine with this.

**Review by hober — APPROVED** [review]
Nice.

File: index.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 We consider privacy a core design principle and differentiator for the web platform (see: [Ethical Web Principles](https://www.w3.org/TR/ethical-web-principles/#privacy), [Unsanctioned Web Tracking](https://www.w3.org/2001/tag/doc/unsanctioned-tracking/), [Private Browsing Modes](https://www.w3.org/2001/tag/doc/private-browsing-modes/), [Privacy Principles](https://www.w3.org/TR/privacy-principles/), [Security & Privacy Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/)).
 
-Many browsers have restricted and/or deprecated third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/)). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG calls for all browsers to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```
Comment (jyasskin) [inline index.html:53]: I think this was a correct use, but I don't object to removing it either. Specifically, the browsers have told folks that 3p cookies are no longer a recommended technology, but they're not removed yet, so they are in fact deprecated.

**Review by jyasskin — APPROVED** [review]

**Pull request merged**